src/scripts/tray.js

```diff
@@ -87,23 +87,34 @@ function createAppTray () {
     };
 }
 
+function setImage (title) {
+    if (title === '•') {
+        title = "Dot";
+    } else if (!isNaN(parseInt(title)) && title > 9) {
+        title = "9Plus";
+    }
+
+    const _iconPath = path.join(__dirname, 'images', icons[process.platform].dir, `icon-tray${title}.png`);
```

Comment:
This will only work for linux now that you have deleted the windows icons. If the icons are the same then you should just remove the platform dir, and put them in a parent folder.